content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded) for the server to become available
+before sending the request.
+
+This allows for robust batch workflows as transient server problems (including
+startup) won't cause job failures.
+
+When an RPC is created while the channel is not in a READY state, without
+waitForReady it will immediately return a failure; with waitForReady it will
+simply block until the connection becomes ready.
+
+For detailed semantics see: [grpc/doc/wait-for-ready.md][grpc doc]
```

Comment:
Maybe just "For detailed semantics, see this." and link the word "this"? It's not clear what significance the current link text has.

Reply:
done